pootle/core/decorators.py

```diff
@@ -132,10 +132,8 @@ def set_resource(request, path_obj, dir_path, filename):
         resource_path = resource_path + filename
 
         try:
-            store = Store.objects.select_related(
-                'translation_project',
-                'parent',
-            ).get(pootle_path=pootle_path)
+            store = Store.objects.live() \
```

Comment:
Whether `select_related()` is necessary for the default manager or not, but that's out of the scope of this PR.